doc/manual/manual.tex

```diff
@@ -10666,6 +10667,71 @@ \subsubsection{Material models}
+
+
+Thermodynamically self-consistent material models must obey the explicit and implicit relations between the different properties \emph{at all pressures and temperatures}:
+
+\paragraph{Consistency in $\boldsymbol{\rho}$-$\boldsymbol{\alpha}$ and $\boldsymbol{\rho}$-$\boldsymbol{\beta_T}$}
+Using the chain rule to combine~\eqref{eq:mm_density},~\eqref{eq:mm_alpha_g} and~\eqref{eq:mm_betaT_g} yields the more familiar definitions of $\alpha$ and $\beta_T$:
+\begin{eqnarray}
+  \alpha &=& -\frac{1}{\rho} \pddc{\rho}{T}{p} \label{eq:mm_thermal_expansivity} \\
+  \beta_T &=& \frac{1}{\rho} \pddc{\rho}{p}{T} \label{eq:mm_isothermal_compressibility}
+\end{eqnarray}
+
+\paragraph{Isobaric heat capacity}
+
+We start by taking the partial derivative of the isobaric heat capacity~\eqref{eq:mm_isobaric_heat_capacity} with respect to pressure at constant temperature:
+\begin{eqnarray}
+  \pddc{C_p^{*}}{p}{T} &=& -T \pddcbc{\mathcal{G}^{*}}{T}{p} \\
+  &=& -T\pddc{\left(\alpha / \rho \right)}{T}{p} \label{eq:heat_capacity_p_dependence}
+\end{eqnarray}
+From this expression it becomes clear that if $\alpha / \rho$ has any temperature dependence, the heat capacity \emph{cannot} be globally constant. One way to solve this issue is to define heat capacity at constant pressure, and then integrate~\eqref{eq:heat_capacity_p_dependence} with respect to pressure:
+\begin{equation}
+  C_p^{*}(p, T) = C_p^{*}(p_{\textrm{ref}}, T) + \int_{p_{\textrm{ref}}}^p \pddc{C_p^{*}}{p}{T} dp
+\end{equation}
+
+\paragraph{Isentropic gradient}
+
+The isentropic gradient is defined as the change in temperature with pressure at constant entropy. Using the cyclic relation, we can define this in terms of partial differentials of the entropy with respect to pressure and temperature:
+\begin{eqnarray}
+  \pddc{T}{p}{S} &=& - \pddc{T}{S}{p} \pddc{S}{p}{T} \\
+  &=& - \left( \frac{T}{C_p^{*}} \right) \left( - \frac{\alpha}{\rho} \right) \\
+  &=& \frac{\alpha T}{\rho C_p^{*}} \label{eq:mm_isentropic_gradient}
+\end{eqnarray}
+
+
+For complex material models, obtaining analytical functions which obey all these relations may be a non-trivial exercise. Indeed, there are many equations of state which exist in the thermodynamic literature which do no obey these relations! 
+
+\emph{A final note of warning: some compressible formulations in \aspect{} (Section~\ref{sec:mass-conservation-approximation}) use the isothermal compressibility, while others use the isentropic compressibility. Fully self-consistent material models must have a switch so that they use the correct compressibility. Conversion between the compressibilities can be achieved using ~\eqref{eq:mm_isentropic_compressibility}.}
```

Comment:
--> must either specify what approximation of the compressible equations they are consistent with (see Section~\ref{sec:approximate-equations}), or have a switch so that they use the correct compressibility for each of the different approximations.